svelte.config.js

```diff
@@ -26,7 +26,11 @@ const config = {
 		}
 	},
 
-	preprocess: [vitePreprocess()]
+	preprocess: [
+		preprocess({
+			postcss: true
```

Comment:
Hey @ghostdevv I've been using `vitePreprocess` is there advantages to using `svelte-preprocess` over the Vite one?

Reply:
`vitePreprocess` is better but it doesn't seem to like the `:` in the tailwind classes in style blocks, and I don't know any other way to fix it